desktop/core/src/desktop/settings.py

```diff
@@ -757,6 +759,30 @@ def disable_database_logging():
     },
   }
 
+  if desktop.conf.TASK_SERVER_V2.ENABLED.get():
```

Comment:
If these methods are only initialised when task server is enabled, then do make sure they are not reference in non-task server Hue setup which can throw error.

Also why this code section in desktop/settings? Would it make sense to shift it in some task server dedicated file or near such methods?